poetry.lock

```diff
@@ -66,14 +66,6 @@ typed-ast = ">=1.4.0"
 [package.extras]
 d = ["aiohttp (>=3.3.2)", "aiohttp-cors"]
 
-[[package]]
```

Comment:
Seems that the lockfile was seriously broken in the master, so good that you fixed that (I assume it was somewhat automatic as soon as you did `poetry add pyyaml`.